packages/server/src/WalletAPIServer.ts

```diff
@@ -103,9 +120,12 @@ export class WalletAPIServer extends RpcNode<
   constructor(
     transport: Transport,
     config: ServerConfig,
-    logger: Logger = defaultLogger,
+    {
+      logger = defaultLogger,
+      customHandlers = {},
+    }: WalletAPIServerOptions = {},
```

Comment:
@ComradeAERGO Should I remove the breaking changes on the `WalletAPIClient` and `WalletAPIServer` constructors ?

I'm not a fan of breaking the API without a major but I don't really want to release a major just for this.

I could keep it non breaking for now and prepare another PR when we want to break the API and do a major with other changes like for the transport layer, wdyt ?

Reply:
I agree with a 2 step solution. Go ahead 👍

Reply:
Done